### ISSUE TITLE
Causing Mac OS X to crash (even log out of account)

### ISSUE THREAD

**Issue by bmorris3** [spacetelescope/jwst_gtvt#30]
I'm trying to run the example code: 
```
$ jwst_gtvt 16:52:58.9 02:24:03
```
The script begins running, returns a bunch of numbers to the terminal, and then promptly crashes my OS. The screen goes black and logs me out of my account. 

I'm running Mac OS X 10.14.6 and Python 3.7.3. Here's my `conda info`: 
```
     active environment : base
    active env location : //anaconda3
            shell level : 1
       user config file : /Users/brettmorris/.condarc
 populated config files :
          conda version : 4.7.12
    conda-build version : 3.18.8
         python version : 3.7.3.final.0
       virtual packages :
       base environment : //anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : //anaconda3/pkgs
                          /Users/brettmorris/.conda/pkgs
       envs directories : //anaconda3/envs
                          /Users/brettmorris/.conda/envs
               platform : osx-64
             user-agent : conda/4.7.12 requests/2.22.0 CPython/3.7.3 Darwin/18.7.0 OSX/10.14.6
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

**Comment by PatrickOgle** [comment]
I ran the same command in my astroconda environment on OSX10.13.6 with Python 3.5.2,
and it ran with no trouble.  A bunch of numbers were returned to the terminal (last line:
2021-09-10    91.44  98.28    91.41  98.25   228.92 235.76    90.87  97.71    96.45 103.29    90.19  97.03), then a matplotlib 'Figure 1' window popped up with 6 available position angle plots.
This might suggest that the problem is associated with the plotting or graphics window.
Do you have similar trouble running other application that utilize matplotlib 'figure'?

**Comment by mfixstsci** [comment]
Hey @bmorris3! I just went ahead and performed the following recipe using the following conda version: `conda version : 4.8.2` with `python version: 3.7.4`

`$ git clone git@github.com:spacetelescope/jwst_gtvt.git`
`$ cd jwst_gtvt`
`$ easy_install .`
`$ jwst_gtvt 16:52:58.9 02:24:03`

and the code ran as expected and produced the figure. Maybe try upgrading your conda package and reinstalling the tool.

**Comment by skendrew** [comment]
I just tried running it a couple of times and had the same result as @bmorris3 . I have the same version of Python and conda version 4.7.10. I'll try updating but can this bug be fixed or at least documented?

**Comment by skendrew** [comment]
@mfixstsci  Your suggested command sequence doesn't work btw - I tried updating conda, reinstalling etc but same result. It sounds like this is a bug in MacOS https://bugs.python.org/issue37833

**Comment by jhunkeler** [comment]
Yeah, this is an operating system bug for sure. We can document it, however there's nothing we can do from our end to fix the problem. 

https://github.com/matplotlib/matplotlib/issues/14999

^Suggests downgrading the `python` and `tk` packages to a particular release temporarily fixed it for them. I don't have 10.14 in front of me at the moment so I cannot confirm.

**Comment by skendrew** [comment]
update: downgrading my python installation from 3.7.3 to 3.7.0 fixes the issue for me.

**Comment by nluetzge** [comment]
Hey @skendrew I just had another case in the Sweden workshop encountering this issue. She downloaded and installed your branch but did not get any error when she installed it with Python 3.7.5 ... Is that whats intended? I thought the fix was to not allow this installation combination anymore?

**Comment by skendrew** [comment]
Hi @nluetzge - yes the setup.py file includes a line that the python version needs to be <= 3.7.0. that works if you install the GTVT using python setup.py install, but I think it won't work if you use other ways to install like pip. Do you know how she tried to install?

**Comment by nluetzge** [comment]
Yea she was using pip.

**Comment by mfixstsci** [comment]
Hey All -- We just released `v0.2.0` and now have unit tests in place to catch these kinds of problems. I am going to go ahead and close this issue.
